src/main/kotlin/com/cognifide/gradle/aem/tooling/clean/Cleaner.kt

```diff
@@ -139,16 +140,12 @@ class Cleaner(project: Project) {
     private fun cleanDotContents(root: File) {
         if (root.isDirectory) {
             aem.project.fileTree(root, filesDotContent).forEach { cleanDotContentFile(it) }
-        } else {
-            cleanDotContentFile(root)
```

Comment:
why? root could be `/content/example/jcr:content` ....

Reply:
for instance if root = /apps/myapp/component/text/text.html, text.html is not a directory obviously, will be removed what is not a expected outcome